README.md

```diff
@@ -33,14 +40,25 @@ Thank you for your interest in contributing to our open-source project! To add y
      }
    },
    ```
-   `Note:` Copy the above code, edit your details, and paste it in the middle (not at the top or bottom). It doesn't matter where you add your profile, it will appear randomly on the website.
-7. Save the changes to the `Profile.json` file.
-8. Create a new branch for your profile `git checkout -b add-profile`
-9. Commit your changes `git commit -m "Add profile"`
-10. Push to the branch `git push origin add-profile`
+   **`Note:`** Copy the above code, edit your details, and paste it in the middle (not at the top or bottom). It doesn't matter where you add your profile, it will appear randomly on the website.
+7. Save the changes to the **`Profile.json`** file.
+8. Create a new branch for your profile **`git checkout -b add-profile`**
+9. Commit your changes **`git commit -m "Add profile"`**
```

Comment:
```suggestion
9. Commit your changes **`git commit -m "add: <your-name>"`**
```

Reply:
@the-amazing-atharva All users are opening PR with same title `Add Profile`, so we decided to change the commit message. Make changes and request for review.